helm/designate-certmanager-webhook/templates/deployment.yaml

```diff
@@ -71,16 +71,33 @@ spec:
             - name: https
               containerPort: 8443
               protocol: TCP
+          {{- if .Values.livenessProbe }}
+          livenessProbe: {{ toYaml .Values.livenessProbe | nindent 12 }}
+          {{- else }}
           livenessProbe:
```

Comment:
Instead of the `if`/`else`, can we please move this over to `values.yaml` and have it as a default for `livenessProbe`/`readinessProbe`/`startupProbe`. That would keep the template clean.

Reply:
@baurmatt thanks for your review.
I've added `probes` property to the `values.yaml` file with some suggested comments to clearly guide user on how to set these probes.
However, I'd like to keep using these `if` / `else` statements to keep the chart be as backward compatible as possible.
Should we keep it?

Reply:
Thanks! :) You can just copy the configuration/values of the `else` block over to the values.yaml as default parameters. This way there shouldn't been any backwards incompatibles.